File: 2016년.js
function solution(a, b) {
    var answer = '';
    const days = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"]
    const months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    let sum = 0;
    for (let i = 0; i < a-1; i++){
        sum += months[i];
    }
    sum += b;
    answer = days[sum % 7]
    return answer;
}

File: 2016년.py
def solution(a, b):
    answer = ''
    months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    days = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"]
    day = (sum(months[:a-1]) + b) % 7
    print(day)
    answer = days[day]
    return answer
